tests/bsim/bluetooth/mesh/tests_scripts/suspend/suspend_disable_resume.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# Copyright 2023 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+source $(dirname "${BASH_SOURCE[0]}")/../../_mesh_test.sh
+
+# Test that periodic publication is stopped when suspending Mesh and disabling
+# Bluetooth, and that it is started again when Bluetooth is re-enabled and Mesh is resumed.
+# The test will fail if no publication is received, or if a publication is received when Mesh
```

Comment:
Suggestion to provide clarity:  This test will fail under two conditions: if no publications are received while the Mesh is enabled, or if a publication is received while Mesh is disabled.

Reply:
Clarified! Thanks for input!